src/main/java/pl/wedding/management/model/reservation/Reservation.java

```diff
@@ -0,0 +1,29 @@
+package pl.wedding.management.model.reservation;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Reservation {
```

Comment:
Co powiesz na to, żeby do encji dodawać końcówkę **Entity**? W ten sposób od razu zaznaczymy odpowiedzialność klasy. Ułatwi to czytanie kodu.